pkg/net/libp2p/libp2ptransport.go

```diff
@@ -0,0 +1,283 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package libp2p
```

Comment:
Maybe add here a `//go:generate go run ./gen/gen.go` directive to re-generate the cbor_gen automatically with `go generate`?